metrics/github/cli.py

```diff
@@ -75,15 +77,15 @@ def pr_throughput(prs, org):
     start = min([pr["created"] for pr in prs])
     days = list(iter_days(start, date.today()))
 
-    with TimescaleDBWriter(GitHubPullRequests) as writer:
-        for day in days:
-            valid_prs = drop_archived_prs(prs, day)
+    for day in days:
+        valid_prs = drop_archived_prs(prs, day)
+
+        merged_prs = [pr for pr in valid_prs if pr["merged"] and pr["merged"] == day]
+        log.info("%s | %s | Processing %s merged PRs", day, org, len(merged_prs))
+        processed_prs = list(iter_prs(merged_prs, day, name="prs_merged"))
```

Comment:
Ditto my comment above about throwing away most of the results.

Reply:
Same as above, fixed in a future commit 👍